cmd/src/cli/js-tests-scaffold/index.js

```diff
@@ -1,21 +1,45 @@
 // This test file uses the tape testing framework.
 // To learn more, go here: https://github.com/substack/tape
 const test = require('tape');
-const Container = require('@holochain/holochain-nodejs');
+const { ConfigBuilder, Container } = require('@holochain/holochain-nodejs');
 
-// instantiate an app from the DNA JSON bundle
-const app = Container.instanceFromNameAndDna("app", "dist/bundle.json")
+// IIFE to keep config-only stuff out of test scope
+const config = (() => {
+  const agentAlice = ConfigBuilder.agent("alice")
 
-// activate the new instance
-app.start()
+  const dna = ConfigBuilder.dna(dnaPath)
+
+  const instanceAlice = ConfigBuilder.instance(agentAlice, dna)
+
+  return ConfigBuilder.container(instanceAlice)
+})()
+
+// Initialize the Container
+const container = new Container(config)
+container.start()
+
+// This function is a bit of temporary boilerplate to construct a convenient object
```

Comment:
I don't believe that we should use this in the scaffold file, better to take it out and use an unwrapped version of 'call'  further down in the code